### PR TITLE
doc: Remove stray \_ escapes in security.md

### PR DESCRIPTION
**Pull request by sparkiegeek**

### CODE REVIEW

File: doc/security.md [hunk 1]
```diff
@@ -236,11 +236,11 @@ instance is using, or can be added to individual instances, as shown below.
...
-Key                      | Type      | Default           | Required  | Description
-:--                      | :--       | :--               | :--       | :--
-security.mac\_filtering  | boolean   | false             | no        | Prevent the instance from spoofing another's MAC address
-security.ipv4\_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv4 address (enables mac\_filtering)
-security.ipv6\_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv6 address (enables mac\_filtering)
+Key                     | Type      | Default           | Required  | Description
+:--                     | :--       | :--               | :--       | :--
+security.mac_filtering  | boolean   | false             | no        | Prevent the instance from spoofing another's MAC address
+security.ipv4_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv4 address (enables mac\_filtering)
+security.ipv6_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv6 address (enables mac\_filtering)
```
Comment (stgraber) [inline doc/security.md:243]: Those need to be escaped as they aren't inside a code block.

Not all markdown parsers are picky about this, but some definitely are :)

**Comment by sparkiegeek** [comment]
https://lxd.readthedocs.io/en/latest/security/ show up in HTML there...

**Comment by stgraber** [comment]
It doesn't in the table here, only does in the text below

File: doc/security.md (hunk 1 above)
Comment (sparkiegeek) [reply doc/security.md]: Ah, right only need to remove them from the ones outside the table. Done - thanks!

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
Thanks!

**Pull request merged**